src/components/Burger/Burger.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import burgers from '../../mocks/burger.json'
+import Product from '../Products/Product'
+import PropTypes from 'prop-types'
+import Headline from '../Headline/Headline'
+
+
+export default function Burger(){
+    return(
+        <>
+        <Headline headline="Burgers" />
+        {burgers.map((burger, index) => (
+        <Product key={index} title={burger.title} image={burger.image} />
+        ))}
+        </>
+    )
+}
+
+Burger.propTypes = { 
+    burgers: PropTypes.array,
```

Comment:
You can also add the contents of the array in the proptypes, if you like. Use arrayOf(PropTypes.object).

Are the props required? If so, add ".isRequired"